### ISSUE TITLE
Feature request: GitHub notifications as a feed

### ISSUE THREAD

**Issue by vikanezrimaya** [snarfed/granary#161]
Adding GitHub notifications to my Notifications channel in Monocle would be nice. Could you add this?

**Comment by snarfed** [comment]
good idea! this would be via the [notifications API](https://developer.github.com/v3/activity/notifications/), which granary doesn't currently use, but should be straightforward to add. PRs are welcome, feel free to take a stab at it! [setup instructions are here.](https://granary.readthedocs.io/en/latest/index.html#development), happy to help if you have trouble.
